tailwind.config.js

```diff
@@ -1,8 +1,20 @@
 module.exports = {
-  mode: "jit",
+  mode: 'jit',
   theme: {
     container: {
       center: true,
     },
+    extend: {
+      fontFamily: {
+        display: ['EB Garamond', 'Hind'],
+      },
+      colors: {
+        primaryLight: '#138D7B',
+        primaryDark: '#40504F',
+        secondary: '#23CEA0',
+        accent: '#EF6B3',
+        black: '#1D1D1D',
```

Comment:
This looks good, but we can leverage some tailwind stuff here

try this pattern

```
colors: {
  primary: {
    light: '#138d7b1,
    dark: '#4050f',
  },